### ISSUE TITLE
PODS disables Varnish

### ISSUE THREAD

**Issue by OceanWind** [pods-framework/pods#5589]
## Issue Overview

PODS has a conflict with server-side caching.

## Expected Behavior

 

Caching would not be disabled because I am using PODS.

## Current Behavior

 

I'm hosting a PODS-enabled site at DreamHost, using their optimized-for-WordPress DreamPress service. This service uses Varnish for caching. I have a site that is "crawling" now, and am trying to debug the issue.

All roads lead to PODS. When running a check on Varnish caching, I am given a warning about PODS:

Plugin: Pods | This plugin uses sessions, which conflicts with server side caching. (Active)

I've tried using PODS alternative cache, but this does not help.

## Steps to Reproduce (for bugs)

1. Enable PODS on a server with Varnish caching
2. Create pages using PODS
3. Watch all the performance gains made by the caching disappear
4.

## Possible Solution

I'm not sure. Make creating session variables optional if you're not using user tracking features...?

## WordPress Environment

<details>
```
WordPress Version: 5.3.2

PHP Version: 7.2.24

MySQL Version: 5.6.34

Server Software: Apache

Your User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36

Session Save Path: /tmp

Session Save Path Exists: Yes

Session Save Path Writeable: Yes

Session Max Lifetime: 1440

Opcode Cache:

Apc: No
Memcached: No
OPcache: No
Redis: No
Object Cache:

APC: No
APCu: No
Memcache: No
Memcached: Yes
Redis: No
WPDB Prefix: wp_jux64i_

WP Multisite Mode: No

WP Memory Limit: 40M

Current Memory Usage: 92.931M

Current Memory Usage (real): 46.500M

Pods Network-Wide Activated: No

Pods Install Location: /home/wp_iu66fp/sacredfire.foundation/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: Sacred Fire Foundation (Divi child)

Currently Active Plugins:

Ajax Load More: 5.1.7.2
Better Search Replace: 1.3.3
Bloom: 1.3.10
Caldera Forms: 1.8.9
Classic Editor: 1.5
DreamHost Panel Login: 1.0.0
Duplicate Post: 3.2.4
Feed Them Social - for Twitter feed, Youtube, Pinterest and more: 2.8.2
InfiniteWP - Client: 1.9.4.5
iThemes Security: 7.6.1
Jetpack by WordPress.com: 8.1.1
Monarch Plugin: 1.4.12
Pods - Custom Content Types and Fields: 2.7.16.2
Pods Alternative Cache: 2.0.6
Proxy Cache Purge: 4.8.1
Recent Posts Widget Extended: 0.9.9.7
Redirection: 4.6.2
UpdraftPlus - Backup/Restore: 1.16.21
User Role Editor: 4.52.2
User Switching: 1.5.3
VaultPress: 2.1.1
Widget Options: 3.7.4
Wordfence Security: 7.4.5
WP Disable Comments: 0.4
WP Google Maps: 8.0.15
Yoast SEO: 12.9.1
```
</details>

## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```

## Workaround or Alternate Solution Until Bug is Addressed

Don't know of any.

**Comment by JoryHogeveen** [comment]
@sc0ttkclark 
Probably related to this issue: https://github.com/pods-framework/pods/issues/4636

**Comment by sc0ttkclark** [comment]
Fixed via #5626
